console/console-init/ui/src/modules/project-detail/components/ProjectCertificateList/CertificateCard.tsx

```diff
@@ -0,0 +1,147 @@
+import { IProjectCertificate } from "./";
+
+export interface ICertificateCardProps {
+  certificate: IProjectCertificate;
+  setOnEditMode: React.Dispatch<React.SetStateAction<boolean>>;
+  id: string;
+}
+
+const styles = StyleSheet.create({
+  row_margin: {
+    marginBottom: 5
+  },
+  float_right: {
+    float: "right"
+  }
+});
+
+export const CertificateCard: React.FunctionComponent<ICertificateCardProps> = ({
+  certificate,
+  setOnEditMode,
+  id
+}) => {
+  const handleEditCertificate = () => {
+    setOnEditMode(true);
+  };
+
+  const handleDeleteCertificate = () => {
+    // TODO: Mutation to delete the certificate
+  };
+
+  const dropdownItems = [
+    <DropdownItem
+      id={`cc-dropdown-edit-${id}`}
+      key="edit"
+      aria-label="Edit certificate"
+      onClick={handleEditCertificate}
+    >
+      Edit
+    </DropdownItem>,
+    <DropdownItem
+      id={`cc-dropdown-delete-${id}`}
+      key="delete"
+      aria-label="delete"
+      onClick={handleDeleteCertificate}
+    >
+      Delete
+    </DropdownItem>
+  ];
+
+  const rowMargin: string = css(styles.row_margin);
+
+  return (
+    <Card id={`cc-card-${id}`}>
+      <CardHeader id={`cc-dard-header-${id}`}>
+        <CardActions className={css(styles.float_right)}>
+          <DropdownWithKebabToggle
+            isPlain={true}
+            dropdownItems={dropdownItems}
+            id={`cc-dropdown-with-kebab-${id}`}
+          />
+        </CardActions>
+      </CardHeader>
+      <CardBody>
+        <Grid>
+          <GridItem span={2}>
+            <Title headingLevel="h1" size="md" id={`cc-subject-dn-title-${id}`}>
+              <b>{getCertificateLabelByKey("subject-dn")}</b>
+            </Title>
+          </GridItem>
+          <GridItem span={10} className={rowMargin}>
+            {certificate["subject-dn"]}
+          </GridItem>
+          <GridItem span={2}>
+            <Title headingLevel="h1" size="md" id={`cc-public-key-title-${id}`}>
+              <b>{getCertificateLabelByKey("public-key")}</b>
+            </Title>
+          </GridItem>
+          <GridItem span={10} className={rowMargin}>
+            {certificate["public-key"]}
+          </GridItem>
+          <GridItem span={2}>
+            <Title
+              headingLevel="h1"
+              size="md"
+              id={`cc-auto-provision-title-${id}`}
+            >
+              <b>{getCertificateLabelByKey("auto-provisioning-enabled")}</b>
+            </Title>
+          </GridItem>
+          <GridItem span={10} className={rowMargin}>
+            <Switch
+              id={`cc-auto-provision-switch-${id}`}
+              label="Enabled"
+              labelOff="Disabled"
+              readOnly
+              isChecked={certificate["auto-provisioning-enabled"]}
+            />
+          </GridItem>
+          <GridItem span={2}>
+            <Title headingLevel="h1" size="md" id={`cc-algorithm-title-${id}`}>
+              <b>{getCertificateLabelByKey("algorithm")}</b>
```

Comment:
We have aleady one fuction for same purpose in top level utils i.e. getLabelByKey. You can use that and add missing key level as you need.

Reply:
done